modules/install-osp-deploy-dualstack.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: REFERENCE
+[id="install-osp-deploy-dualstack_{context}"]
+= Deploying the dual-stack cluster
+
+To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of `install-config.yaml` file.
+
+.Example `install-config.yaml`
+
+[source, yaml]
+----
+apiVersion: v1
+baseDomain: mydomain.test
+featureSet: TechPreviewNoUpgrade <1>
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+metadata:
+  name: mycluster
+networking:
+  machineNetwork: <2>
+  - cidr: "192.168.25.0/24"
+  - cidr: "fd2e:6f44:5dd8:c956::/64"
+  clusterNetwork: <2>
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd01::/48
+    hostPrefix: 64
+  serviceNetwork: <2>
+  - 172.30.0.0/16
+  - fd02::/112
+platform:
+  openstack:
+    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <3>
+    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <4>
+    controlPlanePort:
```

Comment:
It would be beneficial to add a note about `controlPlanePort` field.
It contains detail about the user provided dual-stack network that will be used by all the OpenShift nodes.